docker-images/kafka/cruise-control-scripts/cruise_control_run.sh

```diff
@@ -22,12 +22,38 @@ if [ "x$LOG_DIR" = "x" ]; then
   export LOG_DIR="/tmp"
 fi
 
+echo "$KAFKA_LOG4J_OPTS"
+if [ -z "$KAFKA_LOG4J_OPTS" ]; then
+  export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$CRUISE_CONTROL_HOME/custom-config/log4j.properties"
+fi
+
+# enabling Prometheus JMX exporter as Java agent
+if [ "$KAFKA_METRICS_ENABLED" = "true" ]; then
+  export KAFKA_OPTS="${KAFKA_OPTS} -javaagent:$(ls $KAFKA_HOME/libs/jmx_prometheus_javaagent*.jar)=9404:$CRUISE_CONTROL_HOME/custom-config/metrics-config.yml"
+fi
+
+# enabling Tracing agent (initializes Jaeger tracing) as Java agent
+if [ "$STRIMZI_TRACING" = "jaeger" ]; then
+  export KAFKA_OPTS="$KAFKA_OPTS -javaagent:$(ls $KAFKA_HOME/libs/tracing-agent*.jar)=jaeger"
+fi
```

Comment:
not sure about tracing as I mentioned above. CC is not part of the messages flow for which we want to trace.

Reply:
You are right, I'll strip this out